catalog.html

```diff
@@ -0,0 +1,322 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Седона</title>
+</head>
+
+<body>
+  <header class="page-header">
+    <nav class="navigation">
+      <img src="logo.svg" alt="Город Седона." width="40" height="30">
+      <ul class="navigation-list">
+        <li class="navigaton-item">
```

Comment:
```suggestion
        <li class="navigation-item">
```

дальше поправить аналогичную орфографическую ошибку